src/main/python/camdkit/framework.py

```diff
@@ -364,6 +773,32 @@ def make_documentation(cls) -> dict:
         "description" : desc.__doc__,
         "constraints" : desc.validate.__doc__,
         "sampling" : str(desc.sampling.value),
-        "units": desc.units
+        "units": desc.units if hasattr(desc, "units") else "None"
       })
     return doc
+  
+  """
```

Comment:
`ParameterContainer` is not a temporal container, but a container of metadata parameters. What is this iteration intended to do?

Reply:
This is how I achieve the rearranging of the data for display as per the comment above:
```
 # from:
   obj: {
      param1: [a, b, c]
   }
   # to:
   [{ obj: { 
      param1: a
   }, obj: {
      param1: b
   }, obj: {
      param1: c
   }]
```

Reply:
A `ParameterContainer` can only be iterated on if it consists only of `REGULAR` parameters and only if all such parameters are sampled at the same rate. I recommend removing this method and leaving to the application.

Reply:
I don't currently have an alternative way of doing this. Can we discuss?